### PR TITLE
Use Azure Artifacts instead of MyGet

### PR DESCRIPTION
**Pull request by garath**
As part of [moving to Azure Artifacts from MyGet](https://github.com/dotnet/arcade/blob/master/Documentation/MigrationPlan/MyGetFeeds.md), point all MyGet feed sources to new Azure Artifacts feed.

@dougbu FYI, from email.

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -2,10 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <!--
-      Restore sources should be defined in build/sources.props.
```
Comment (dougbu) [inline NuGet.config:6]: This comment is still correct

File: NuGet.config [hunk 2]
```diff
@@ -2,10 +2,6 @@
...
-      The only allowed feed here is myget.org/aspnet-tools which is required to workaround https://github.com/Microsoft/msbuild/issues/2914
-    -->
-    <add key="myget.org aspnetcore-tools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <add key="myget.org aspnetcore-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
```
Comment (dougbu) [inline NuGet.config:5]: Please rename this key to match the actual feed e.g. `myget-legacy internal`.

File: NuGet.config (hunk 1 above)
Comment (garath) [reply NuGet.config:6]: Restored in commit 5c566c625.

File: NuGet.config (hunk 2 above)
Comment (garath) [reply NuGet.config]: Renamed in commit 5c566c625.

**Comment by markwilkie** [comment]
Any chance you could merge @dougbu ?

**Pull request merged**